.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+on: [push, pull_request]
```

Comment:
let's clean this using only pull request and release events and add the artifact to the releases published